src/RoadRegistry.BackOffice/Core/RoadNetworkSnapshotReaderWriter.cs

```diff
@@ -105,5 +105,34 @@ public async Task WriteSnapshot(Messages.RoadNetworkSnapshot snapshot, int versi
                 }
             }
         }
+
+        public async Task SetHeadToVersion(int version, CancellationToken cancellationToken)
+        {
+            var newSnapshotBlobName = SnapshotPrefix.Append(new BlobName(version.ToString()));
+            if (!await _client.BlobExistsAsync(newSnapshotBlobName, cancellationToken))
+            {
+                throw new Exception();
```

Comment:
don't throw System.Exception, prefer InvalidOperationException

Reply:
Fixed in https://github.com/Informatievlaanderen/road-registry/pull/539